client/src/js/services/ScrollService.js

```diff
@@ -21,14 +21,16 @@ ScrollService.$inject = ['$location', '$anchorScroll', '$timeout'];
  */
 function ScrollService($location, $anchorScroll, $timeout) { 
   var scrollDelay = 0;
-
+  
   // Always scroll an additional 50 pixels to ensure element is visiable 
   $anchorScroll.yOffset = 50;
 
   /** 
+   * @deprecated
```

Comment:
Sure! I will do this after the Notify service has landed in `master` - notification patterns will significantly inform how this scrolling is used.